src/main/java/org/kairosdb/core/http/rest/MetricsResource.java

```diff
@@ -91,6 +93,27 @@
 
 	private final KairosDataPointFactory m_kairosDataPointFactory;
 
+	public static final String READ_TIMEOUT = "kairosdb.datastore.datapoints.read.timeout";
+	public static final String READ_THREAD_POOL_MAX_SIZE = "kairosdb.datastore.datapoints.read.threadpool.max.size";
+	public static final String READ_THREAD_POOL_CORE_SIZE = "kairosdb.datastore.datapoints.read.threadpool.core.size";
+	public static final String READ_THREAD_KEEP_ALIVE_TIME = "kairosdb.datastore.datapoints.read.thread.keep.alive.time.milliseconds";
+
+	@com.google.inject.Inject
+	@Named(READ_TIMEOUT)
+	private int m_readTimeout = 20000;
+
+	@com.google.inject.Inject
+	@Named(READ_THREAD_POOL_MAX_SIZE)
+	private int m_readThreadPoolMaxSize = 100;
+
+	@com.google.inject.Inject
+	@Named(READ_THREAD_POOL_CORE_SIZE)
+	private int m_readThreadPoolCoreSize = 10;
+
+	@com.google.inject.Inject
```

Comment:
Do we actually need to explicitly use google's @Inject here over the javax @Inject used everywhere else in the class?